src/wizard/interactive_build.jl

```diff
@@ -503,7 +504,7 @@ function step5c(state::WizardState)
                 verbose=false,
                 tee_stream=state.outs
             )
-
+            prefix == nothing && (ok = false; return)
```

Comment:
I don't really like this syntax; please use the more explicit

```
if prefix == nothing
    ok = false
    return
end
```

It also seems to me that you mean `continue` here, not `return`.

Reply:
I believe you can't continue from inside a closure?

Reply:
Oh, of course, you're right.  I missed the `cd() do`